### DISCUSSION TITLE
Interface jump error

### DISCUSSION THREAD

**Discussion by mengqunhan** [community/community#19912] [Copilot Conversations] [answered]
Hi, 
    When I was using Copliot, I met some problem, or something I thought unreasonable. When Copliot generates a very long piece of code, the VS Code text area will jump to the bottom of generated code, I have to scroll back to find the cursor, this is very inconvenient.

u can see this issue in detail through the video below.

https://user-images.githubusercontent.com/43869691/177022667-bc7b3153-0877-429f-84c2-23e2f053c48a.mp4

**Comment by sardorb3k** [comment, marked as answer]
Open the Command Palette
`Ctrl + Shift + P`
Then type:
`Reload Window`
